### PR TITLE
Add recipe for membrane-curvature

### PR DESCRIPTION
**Pull request by ojeda-e**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ojeda-e** [comment]
@conda-forge-admin, please ping team

**Comment by carterbox** [comment]
@conda-forge/staged-recipes

**Comment by ojeda-e** [comment]
@conda-forge/help-python

File: recipes/membrane-curvature/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - membrane_curvature/tests
+about:
+  home: https://github.com/MDAnalysis/membrane-curvature
+  summary: MDAnalysis tool to calculate membrane curvature from MD simulations.
+  license: GPL-3.0-only
+  license_file: LICENSE
+  doc_url: https://membrane-curvature.readthedocs.io/
+  dev_url: https://github.com/MDAnalysis/membrane-curvature
+extra:
+  recipe-maintainers:
```
Comment (ocefpaf) [inline recipes/membrane-curvature/meta.yaml:42]:
```suggestion
{% set name = "membrane-curvature" %}
{% set version = "1.1.0" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/membrane_curvature-{{ version }}.tar.gz
  sha256: 7d72d15fa9616c335cf0b9f3463fb8cdbcb6578249116eb7783d7c0713a4b827

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
  number: 0

requirements:
  host:
    - python >=3.9
    - setuptools
    - versioningit
    - pip
  run:
    - python >=3.9
    - numpy >=1.20.0
    - mdanalysis >=2.0.0

test:
  imports:
    - membrane_curvature
  commands:
    - pip check
    - pytest
  requires:
    - pip
    - pytest
  source_files:
    - membrane_curvature/tests

about:
  home: https://github.com/MDAnalysis/membrane-curvature
  summary: MDAnalysis tool to calculate membrane curvature from MD simulations.
  license: GPL-3.0-only
  license_file: LICENSE
  doc_url: https://membrane-curvature.readthedocs.io/
  dev_url: https://github.com/MDAnalysis/membrane-curvature

extra:
  recipe-maintainers:
```

**Pull request merged**